anthos-bm-gcp-terraform/docs/manuallb_install.md

```diff
@@ -0,0 +1,299 @@
+...
+[2022-04-06 16:43:07+0000] Waiting for node pools to become ready OK
+[2022-04-06 16:43:37+0000] Moving admin cluster resources to the created admin cluster
+[2022-04-06 16:44:02+0000] Flushing logs... OK
+[2022-04-06 16:44:02+0000] Deleting bootstrap cluster... OK
+
+[+] Anthos on bare metal installation complete!
+[+] Run [export KUBECONFIG=/home/tfadmin/bmctl-workspace/cluster1/cluster1-kubeconfig] to set the kubeconfig
+[+] Run the [/home/tfadmin/login.sh] script to generate a token that you can use to login to the cluster from the Google Cloud Console
+
+[+] Configuring the istio ingress for public access...
+[+] Extracting necessary variables from the init.vars file
+[+] Fetching the NodePort of the istio-ingress service in the cluster
+[+] Firewall ports to be updated: tcp:6444,tcp:443,tcp:30613
+[+] Creating the patch file for istio-ingress at: /tmp/ingress-patch.yaml
+[+] Patching the istio-ingress service with the external ip: 34.111.235.160
+service/istio-ingress patched
+[+] Deploying the Point of Sale application manifests
+configmap/service-configs created
+deployment.apps/api-server created
+deployment.apps/payments created
+deployment.apps/inventory created
+service/api-server-svc created
+service/payments-svc created
+service/inventory-svc created
+[+] Deploying the ingress resource for the Point of Sale application
+ingress.networking.k8s.io/pos-ingress created
+[+] Updating the Network Endpoint Group (NEG) of the Ingress loadbalancer with the controlPlaneIP:nodePort
+[+] Adding network endpoint [instance=cluster1-abm-cp1-001,port=30613] to NEG group [abm-ing-lb-neg]
+Attaching 1 endpoints to [abm-ing-lb-neg]....
+........done.
+[+] Adding network endpoint [instance=cluster1-abm-cp2-001,port=30613] to NEG group [abm-ing-lb-neg]
+Attaching 1 endpoints to [abm-ing-lb-neg]....
+.........done.
+[+] Adding network endpoint [instance=cluster1-abm-cp3-001,port=30613] to NEG group [abm-ing-lb-neg]
+Attaching 1 endpoints to [abm-ing-lb-neg]....
+........done.
+[+] Updating firewall rule allow traffic to the ingress nodePort: tcp:6444,tcp:443,tcp:30613
+Updated [https://www.googleapis.com/compute/v1/projects/shabir-gcp-lb-8/global/firewalls/abm-allow-lb-traffic-rule].
+
+[*] Configuring Ingress Loadbalancer complete!
+[*] You may reach the services exposed by the ingress at: 34.111.235.160
+```
+---
+### Interacting with the Baremetal cluster
+
+#### Controlplane / API Server
+
+You can find your cluster's `kubeconfig` file on the admin machine in the
+`bmctl-workspace` directory _(see logs of the installation for the exact path)_.
+Since the `API Server` of the cluster is exposed via the GCP Loadbalancer, the
+`kubeconfig` file is configured to use the **Public IP Address** of the
+loadbalancer. Thus, you can copy the `kubeconfig` file to your local workstation
+and interact with the cluster using `kubectl`.
+
+To verify your deployment, complete the following steps:
+
+1. Copy the contents of `kubeconfig` file to your workstation.
```

Comment:
i had to set my project id in my workstation, could be worth noting here?

Reply:
[These lines](https://github.com/GoogleCloudPlatform/anthos-samples/pull/218/files#diff-dbd4f6dc96e4f99879f1ec3afb8df3d8024fc08a37098ebb1dae62ec0e5816c3R234-R238) have an instruction for it. Did you still face an issues?

Reply:
I had linked to the lines `234-238` on the manuallb_install.md file. If you click the link and leave it a **few seconds** then it should scroll down to the exact highlighted spot. It ain't working?